src/converters/touch.hpp

```diff
@@ -29,6 +29,7 @@
 */
 #include <naoqi_bridge_msgs/Bumper.h>
 #include <naoqi_bridge_msgs/TactileTouch.h>
+#include <naoqi_bridge_msgs/HandTouch.h>
```

Comment:
sorry for the nicktip: could you move this one line up for alphabetical order ? (same below). Thanks!